src/js/events/change-state-event.ts

```diff
@@ -26,7 +27,14 @@ export default function ChangeStateEvent(model, newstate, oldstate) {
         const type = newstate.replace(/(?:ing|d)$/, '');
         const reason = normalizeReason(newstate, model.mediaModel.get('mediaState'));
 
-        const evt = {
+        const evt: {
+            type: string;
```

Comment:
maybe interface for newstate, oldstate, reason if every event has those fields.

Reply:
i like this idea